projects/plugins/jetpack-mu-wpcom/composer.json

```diff
@@ -3,10 +3,10 @@
 	"description": "Enhances your site with features powered by WordPress.com",
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-or-later",
-	"require": {},
```

Comment:
> Hmm thought it was something to do with the empty changelog, but that didn't seem to help: [c50d89e](https://github.com/Automattic/jetpack/pull/27815/commits/c50d89e1c0a87fefc8675001e62871cdd2ceb398) Any idea on what the snag is, or how I can get more helpful error output (e.g., line number)?

I don't know about getting more helpful output, but I suspect it's because of this removal and the code at https://github.com/Automattic/jetpack/blob/c20c48509c7784e5fcc1763b949306f78eb99455/projects/github-actions/repo-gardening/src/utils/get-affected-changelogger-projects.js#L18

For this PR, try adding this back. In a separate PR, we should change that code to use `?.` for the accesses so it doesn't error out for something like this.

Reply:
>  In a separate PR, we should change that code to use ?. for the access

Nice catch! I'll be sure to follow up on that one 👍